docs/release-notes/v11.2.0.md

```diff
@@ -0,0 +1,121 @@
+
+
+The purpose of [IAM portal](/development_suite/identity-and-access-management/manage-identities.md) is to allow Company Owners - _at Company level_ -, and Project Administrators - _solely at Project level_ -, to view and manage roles directly assigned to identities in order to grant them specific capabilities.
+:::
+
+#### Delete a Provider
+
+In the "Providers" section  – inside the Company Overview area  – you can now remove a provider. 
+This can only be done if the provider is not used in any Project or if it is not set as the default of the respective Company.
+
+For more information, [go to the documentation](/development_suite/set-up-infrastructure/configure-provider.mdx).
+
+### Improvements
+
+#### Project creation and respective permission management 
+
+If the user do not have the necessary permissions to create a Project in a respective Company, the user can not proceed to the step two of creation.
+
+#### Marketplace filtered only for Private Company items
+
+It is now possible to choose to view only private Marketplace items available exclusively for my Company, thanks to a switch button in the top right of the Marketplace interface.   
+For more information about how to set up privatized Marketplace items for Companies, visit the dedicated [documentation page](/marketplace/add_to_marketplace/contributing_overview.md#public-and-private-marketplace).
+
+#### Microservice HTTPS and SSH RepoURLs manageability
+
+From the detail page of a microservice - inside the Design Area - it is possible to configure and update both HTTPS and SSH repository URLs.
+By configuring these fields, the View repository and Clone repository actions will be enabled for that specific microservice.
+
+#### More features in collapsed dynamic sidebar
+
+In the Design Area, it is now possible to view the additional features yet to be activated even when using dynamic sidebar in collapsed mode.
+
+:::info
+Dynamic sidebar can be activated from the [Feature Preview](/development_suite/user-settings/feature-preview.md) modal, available for SaaS Console users.
+:::
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* 
+* 
+
+## Fast Data
+
+### Bug Fix
+
+This version addressed a bug in the No Code ER Schema, that made it impossible to create a rule between two fields with the same name.
+
+#### Support for autogenerated Projection Updates in Real Time Updater v7.5.2
+The Real Time Updater now supports the Projection Updates autogenerated by the console.
+
+#### Ingestion Reloader can now handle very large amount of files
+Fixed the Ingestion Reloader way of downloading files preventing memory overflow. From version 1.3.1 onwards, the files are properly downloaded with streams to prevent memory problems.
+
+#### Ingestion Reloader v1.3.1 invalid date throws an error
+When starting a reingestion the Ingestion Reloader properly checks whether a date is valid or not and throws an error if needed.
```

Comment:
```suggestion
#### Support for autogenerated Projection Updates in Real Time Updater v7.5.2

The Real Time Updater now supports the Projection Updates autogenerated by the console.

#### Ingestion Reloader can now handle very large amount of files

Fixed the Ingestion Reloader way of downloading files preventing memory overflow. From version 1.3.1 onwards, the files are properly downloaded with streams to prevent memory problems.

#### Ingestion Reloader v1.3.1 invalid date throws an error

When starting a reingestion the Ingestion Reloader properly checks whether a date is valid or not and throws an error if needed.
```